packages/hoppscotch-common/src/components/environments/index.vue

```diff
@@ -408,7 +408,6 @@ const selectedEnv = computed(() => {
         teamEnvID: selectedEnvironmentIndex.value.teamEnvID,
       }
     } else {
-      selectedEnvironmentIndex.value = { type: "NO_ENV_SELECTED" }
       return { type: "NO_ENV_SELECTED" }
     }
   } else {
```

Comment:
It is a bad practice to have side effects in computed properties. since we're here already,     can you also remove the `selectedEnvironmentIndex.value = { type: "NO_ENV_SELECTED" }` from the next else, if it doesnt serve any purpose. if it has purpose, we can move it to a watcher